From 255288fc56a12e57a8fe4df6450b4049fec02134 Mon Sep 17 00:00:00 2001
From: Christian Jaeger <ch@christianjaeger.ch>
Date: Sat, 25 Jun 2022 13:37:34 +0100
Subject: [PATCH 146/191] Background orange

But it's not very bright, and there's no better shade (?).
---
 .bashrc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/.bashrc b/.bashrc
index 2bc0707..bc9cc69 100644
--- a/.bashrc
+++ b/.bashrc
@@ -23,7 +23,7 @@ __ps1_show_exitcode () {
     # and for this have to use \001 \002 over \[ \] .
     # (Also see https://mywiki.wooledge.org/BashFAQ/053 .)
     if [ "$exitcode" -ne 0 ]; then
-	echo -ne '\001\033[01;103m\002'
+	echo -ne '\001\033[01;48;5;208m\002'
 	printf '%3d' "$exitcode"
 	echo -ne '\001\033[00m\002'
 	echo -n ' '
-- 
2.47.3

